src/to_dom.js

```diff
@@ -4,7 +4,8 @@
 // itself, or an array.
 //
 // An array describes a DOM element. The first value in the array
-// should be a string—the name of the DOM element. If the second
+// should be a string—the name of the DOM element. It also can be
+// space-separated the XML namespace and the name. If the second
```

Comment:
"the name of the DOM element, optionally prefixed by a namespace URL and a space." is probably a better phrasing.